.circleci/config.yml

```diff
@@ -40,17 +40,25 @@ _refs:
   gus-create-change-mgmt: &gus-create-change-mgmt
     name: 'Gus create change case management'
     command: |
-      npm install -g @salesforce/change-case-management
-      export GUS_BUILD="$(node -pe "require('./packages/salesforcedx-vscode/package.json').version")"
-      echo "Gus build is ${GUS_BUILD}"
-      sfchangecase create -i ${GUS_TEMPLATE_ID} --release offcore.tooling.${GUS_BUILD} --location ${REPO_LOCATION}
-      sfchangecase check --release offcore.tooling.${GUS_BUILD} --location ${REPO_LOCATION}
+      if [ "$GUS_SKIP" == "false" ]; then
```

Comment:
Are we skipping change management creation by default?

Reply:
No, the default for that variable will be false.